### DISCUSSION TITLE
security code review or security audit

### DISCUSSION THREAD

**Discussion by sssipaz** [neos-sdi/adfsmfa#204] [Q&A] [answered]
Hello,

Thanks for such a great product. We started to use in and our InfoSec team asked is there any security code review or security audit was done by well known, trusted third party?

**Comment by redhook62** [comment, marked as answer]
Hello,

Thank you,

No, we have not had a security audit done by a recognized third party.

Unpretentious, we relied on the remarks of the Stargazers.

Be aware, however, that very large companies use it. Without being able to name them, the world leaders in space, aeronautics, transport in France, the world of software, religious associations, International organisations, etc ...

It is, an open-source project, and we do it alongside our work. 
We didn't try to promote it, didn't want to subscribe to GitHub Sponsors ...
But, we have been monitoring and continuing to develop it for 5 years.

regards
